data/state/accountsDB.go

```diff
@@ -856,9 +863,17 @@ func (adb *AccountsDB) SetStateCheckpoint(rootHash []byte) {
 		adb.mainTrie.SetCheckpoint(rootHash)
 		adb.snapshotUserAccountDataTrie(rootHash)
 		adb.mainTrie.ExitSnapshotMode()
+
+		adb.increaseNumCheckpoints()
 	}()
 }
 
+func (adb *AccountsDB) increaseNumCheckpoints() {
+	time.Sleep(time.Duration(adb.mainTrie.GetSnapshotDbBatchDelay()) * time.Second)
```

Comment:
this metric isn't persistent. Do we need it to be?

Reply:
also these metrics might be useful in `/node/status` I guess

Reply:
Yes, will make it persistent. I think that `/node/statistics` is enough for now.

Reply:
I think `/node/status` is the right place for this one  🙏

Reply:
Refactored.